controllers/users.js

```diff
@@ -51,15 +50,7 @@ const getUserById = async (req, res) => {
     return res.boom.notFound("User doesn't exist");
   }
 
-  const { phone = "", email = "", ...user } = result.user;
-  try {
-    user.phone = obfuscate.obfuscatePhone(phone);
-    user.email = obfuscate.obfuscateMail(email);
-  } catch (error) {
-    logger.error(`Error while formatting phone and email: ${error}`);
-    return res.boom.badImplementation("Error while formatting phone and email");
-  }
-
```

Comment:
Please confirm if, the phone(88xxxxxxxx) and email (sommailxxxxxxxx.xx) being removed will break any frontend app, where it's being used